_inc/client/components/jetpack-termination-dialog/single-feature.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'components/gridicon';
+import React from 'react';
+
+const SingleFeature = ( { amount, description, gridIcon, title } ) => {
+	return (
+		<div className="jetpack-termination-dialog__feature">
+			<div className="jetpack-termination-dialog__feature-header">
+				<h3>{ title }</h3>
+				<Gridicon icon={ gridIcon } />
+			</div>
+			<div className="jetpack-termination-dialog__feature-body">
+				<p className="jetpack-termination-dialog__feature-body-amount">{ amount }</p>
```

Comment:
I think we'll need this amount to be internationalized, maybe, so you get something like `2,000` instead of `2000`.

Something like this should do the trick:

```diff
diff --git a/_inc/client/components/jetpack-termination-dialog/single-feature.jsx b/_inc/client/components/jetpack-termination-dialog/single-feature.jsx
index 4b2697e6e..c950164bc 100644
--- a/_inc/client/components/jetpack-termination-dialog/single-feature.jsx
+++ b/_inc/client/components/jetpack-termination-dialog/single-feature.jsx
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import Gridicon from 'components/gridicon';
+import { numberFormat } from 'i18n-calypso';
 import React from 'react';
 
 const SingleFeature = ( { amount, description, gridIcon, title } ) => {
@@ -12,7 +13,7 @@ const SingleFeature = ( { amount, description, gridIcon, title } ) => {
 				<Gridicon icon={ gridIcon } />
 			</div>
 			<div className="jetpack-termination-dialog__feature-body">
-				<p className="jetpack-termination-dialog__feature-body-amount">{ amount }</p>
+				<p className="jetpack-termination-dialog__feature-body-amount">{ numberFormat( amount ) }</p>
 				<p className="jetpack-termination-dialog__feature-body-description">{ description }</p>
 			</div>
 		</div>

```

**Before**

![image](https://user-images.githubusercontent.com/426388/67205950-14143200-f411-11e9-8deb-9e4ccd944932.png)


**After**

![image](https://user-images.githubusercontent.com/426388/67205965-170f2280-f411-11e9-8397-1881a081d9d2.png)